### ISSUE TITLE
ChromeDriver verbose logging

### ISSUE THREAD

**Issue by sutris-sdp** [vvo/selenium-standalone#282]
Hi, may I know how to start a `selenium-standalone` server with ChromeDriver verbose logging ([reference](https://sites.google.com/a/chromium.org/chromedriver/logging))? Thank you 🙏

**Comment by stephenash** [comment]
Does something like the following work for what you are trying to do?

```bash
selenium-standalone start --javaArgs="-Dwebdriver.chrome.logfile=chrome.log"
```

**Comment by sutris-sdp** [comment]
Thank you @stephenash; unfortunate using that `--javaArgs` arg produce the same log as without the arg. 

Maybe this docs could help: https://sites.google.com/a/chromium.org/chromedriver/logging

**Comment by stephenash** [comment]
When I tested `--javaArgs="-Dwebdriver.chrome.logfile=chrome.log"`, it produced a different log file than running without it (located at `./chrome.log` in that example, and not logging to stdout). That was based on the Java example that was listed in the ChromeDriver logging link that you provided.

The other examples that are listed on that page (e.g. `chromedriver.exe --verbose`) don't work for selenium-standalone. It is only starting the selenium.jar process to setup the Selenium hub/server. The selenium.jar process in turn starts Chrome based on how the driver was configured.

I also tried changing how selenium-standalone passes in arguments to where the driver is defined like

```text
-Dwebdriver.chrome.driver="/tmp/selenium-standalone/.selenium/chromedriver/2.29-x64-chromedriver --verbose"
```

but that didn't allow Chrome to start at all. If you have other examples of how this is configured through Selenium server, we can explore it more. Without that, I'm not sure exactly what selenium-standalone could do to support this case.
